app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :post
+
+  def update_likes
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Great work on defining the methods for your like model! 😊 To enhance the maintainability and organization of your code, I recommend considering the use of [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), specifically `after_save`, to automatically trigger the `update_likes` method each time a new like is saved to the database. This approach will help you keep your code clean and ensure that the counter is always up to date without manual intervention.
   - Additionally, to encapsulate the `update_likes` method properly and prevent accidental external calls, consider marking it as private. This encapsulation will improve code readability and adherence to best practices.

Happy coding! 👍👨‍💻🚀